apps/mobile/src/screens/PassengerDetailScreen.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+import * as React from 'react';
+import { PassengerDetail } from '@kiwicom/margarita-core';
+import { View } from 'react-native';
+import { StyleSheet } from '@kiwicom/universal-components';
+import { type NavigationScreenProp } from 'react-navigation';
+
+type NavigationParams = {||};
+
+type Props = {|
+  +navigation: NavigationScreenProp<NavigationParams>,
+|};
+
+export default function PassengerDetailScreen(props: Props) {
```

Comment:
adding props to this functions is what causing `eslint` to fail. Please remove it, props are not used in this component.

Reply:
Hmm wasnt failing on local, removing now